test.js

```diff
@@ -395,9 +394,6 @@ test('test that missing type errors', function (t, ctx, done) {
   var ws = ctx.db.createWriteStream()
   ws.on('error', function (err) {
     t.equal(err.message, '`type` must be \'put\' or \'del\'', 'should error')
-    errored = true
-  })
-  ws.on('close', function () {
```

Comment:
Changed this, since switching to `flush-write-stream` doesn't produce a `'close'` event when doing `.end()`.